### DISCUSSION TITLE
Link in readme

### DISCUSSION THREAD

**Discussion by PiterDev** [maximmax42/Discord-CustomRP#86] [General]
Hey, not sure if this is a bug or maybe something is wrong on my end, so I'm writing here. On the page, there is a hyperlink for people who are already translators in the project, (sends you over [here](https://poeditor.com/projects/view?id=409229)) but when you click it, it gives you a 404 error, because there is no global page for all translators, only for each language I think. A solution might be making the flags send you to a link for each translation page but I don't know if it's possible cause I'm not experienced with markdown.

**Comment by maximmax42** [comment]
Yeah it's possible to make progress badges (that's what you mean by flags I suppose?) link to the language pages. Are you logged in on POEditor btw?
